pages/_app.js

```diff
@@ -185,7 +185,12 @@ export default function App({ Component, pageProps }) {
     setPetCollection((prevPets) =>
       prevPets.map((pet) => {
         if (pet.id === currentPet) {
-          return { ...pet, dying: false, alive: false };
+          return {
+            ...pet,
+            dying: false,
+            alive: false,
```

Comment:
when we have keys or variables that store booleans a their values, they are usually prefixed by is or has, to identify at a first glance, regardless of where you are in your application, that the value will be a boolean.
In this case, is would be a could prefix:
```suggestion
            isDying: false,
            isAlive: false,
```

This will mean some more adaptations in your code base where you are using the keys `dying` and `alive`.